app/src/main/java/com/example/parkingproject/utils/Picker.java

```diff
@@ -37,6 +37,8 @@ public void onDateSet(DatePicker view, int year, int month, int dayOfMonth) {
                     public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
                         calendar.set(Calendar.HOUR_OF_DAY, hourOfDay);
                         calendar.set(Calendar.MINUTE, minute);
+                        @SuppressLint("SimpleDateFormat")
+                        SimpleDateFormat simpleDateFormat = new SimpleDateFormat(FORMAT);
```

Comment:
The warning disabled here helps you to avoid having location problems, many times date/time format will depend on the location (country) of the user. So as recommended use `SimpleDateFormat.getDateTimeInstance()` or do `new SimpleDateFormat(FORMAT, Locale.getDefault())`.